docs/sources/release-notes/v2-8.md

```diff
@@ -0,0 +1,28 @@
+---
+title: V2.8
+description: Version 2.8 release notes
+weight: 55
+---
+
+# V2.8
+Grafana Labs is excited to announce the release of Loki 2.8. Here's a summary of new enhancements and important fixes:
+
+## Features and enhancements
+
+- **TSDB index no longer experimental** After extensive testing of the new TSDB index in our Grafana Cloud Logs offering, we can safely announce that it is no longer experimental and encourage all Loki deployments to use it!
```

Comment:
Great, TSDB is finally ready, I can finally replace Cassandra, operating and maintaining dozens of Cassandra clusters is really painful for me